pyrdp/player/ReplayWindow.py

```diff
@@ -17,15 +18,17 @@ class ReplayWindow(BaseWindow):
     Class for managing replay tabs.
     """
 
-    def __init__(self, options: Dict[str, object], parent: QWidget = None):
-        super().__init__(options, parent)
+    def __init__(self, options: Dict[str, object], mainWindow: MainWindow, parent: QWidget = None):
```

Comment:
Can't we just use the parent QWidget argument here? isn't that the whole point of Qt?  the `ReplayTab` is logically a child of the QWindow. I would test that as it would keep the constructor the same and be more consistent with Qt You can still cache the main window to avoid having to call `self.getParent()`.

Reply:
I indeed found a more intelligent way to handle this, see the latest commit! thanks